### PR TITLE
[1.3.7-prepare#5421] Change the status box text (#5421)

### PR DESCRIPTION
**Pull request by ruanwenjun**
pr #5421

### CODE REVIEW

File: dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js [hunk 1]
```diff
@@ -420,7 +420,7 @@ export default {
   'Process instance details': '流程实例详情',
   'Create Resource': '创建资源',
   'User Center': '用户中心',
-  'Please enter method': '请输入方法',
+  AllStatus: '全部状态',
```
Comment (Technoboy-) [inline dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js]: Missing single quote?

**Review by Technoboy- — CHANGES_REQUESTED** [review]

File: dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js (hunk 1 above)
Comment (ruanwenjun) [reply dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js]: Thanks for your review, it seems the single quote is not required. Both the two kinds of writing exist in the project.

Comment (Technoboy-) [reply dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js]: How about keeping the same with others?

Comment (ruanwenjun) [reply dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js]: Good suggestion

**Review by Technoboy- — APPROVED** [review]
LGTM

**Pull request merged**